src/js/components/form.jsx

```diff
@@ -37,16 +46,14 @@ class Form extends React.Component {
             useOfService: this.getSelectedRadio()
         };
 
-        ( data["email"] ||
-          data["firstName"] ||
-          data["lastName"] ||
-          data["nationality"] ||
-          data["useOfService"] === "" || "undefined") ?
-         alert("Please complete all fields.") : this.submit(data);
+        this.checkInput(data, (result) => {
+            console.log('>>>>>', result);
+            result ? alert("Please complete all fields.") : this.submit(data);
```

Comment:
Is this ternary operator the wrong way round? Currently it's alerting if all fields are filled, and submitting if not